src/components/ChooseYourCocktail.jsx

```diff
@@ -11,15 +12,23 @@ function ChooseYourCocktail({ cocktailList }) {
   const query = useQuery();
   const page = parseInt((query.get('page') ?? 1), 10);
   const pageSize = 3;
+  const cocktailListLength = parseInt(cocktailList.length / 3)+1;
```

Comment:
what if the division is exact? if I have 18 cocktails, I should have 6 pages: I will have 7 :)